component-tools/src/main/java/org/talend/sdk/component/tools/DitaDocumentationGenerator.java

```diff
@@ -98,64 +100,128 @@ protected String emptyDefaultValue() {
             final TransformerFactory transformerFactory, final ZipOutputStream zip,
             final Collection<String> directories) throws ParserConfigurationException {
+
         final String family = componentDescription.getFamily();
         final String name = componentDescription.getName();
         final String componentId = family + '-' + name;
 
         final DocumentBuilder builder = factory.newDocumentBuilder();
         final Document xml = builder.newDocument();
 
-        final Element topic = xml.createElement("reference");
-        topic.setAttribute("id", "connector_" + componentId);
-        topic.setAttribute("xml:lang", ofNullable(getLocale().getLanguage()).filter(it -> !it.isEmpty()).orElse("en"));
-        topic.setAttribute("xtrc", "standard_component");
-        xml.appendChild(topic);
+        final Element reference = xml.createElement("reference");
+        reference.setAttribute("id", "connector_" + componentId);
+        reference.setAttribute("id", "connector-" + family + '-' + name);
+        reference
+                .setAttribute("xml:lang",
+                        ofNullable(getLocale().getLanguage()).filter(it -> !it.isEmpty()).orElse("en-us"));
+        xml.appendChild(reference);
 
         final Element title = xml.createElement("title");
         title.setAttribute("id", "component_title_" + componentId);
         title.setTextContent(name + " parameters");
-        topic.appendChild(title);
+        reference.appendChild(title);
 
         final Element shortdesc = xml.createElement("shortdesc");
         shortdesc.setTextContent(componentDescription.getDocumentation().trim());
-        topic.appendChild(shortdesc);
+        reference.appendChild(shortdesc);
 
         final Element prolog = xml.createElement("prolog");
         final Element metadata = xml.createElement("metadata");
 
-        final Element pageId = xml.createElement("othermeta");
-        pageId.setAttribute("name", "pageid");
-        pageId.setAttribute("content", "component_" + componentId);
-        metadata.appendChild(pageId);
-
         final Element othermeta = xml.createElement("othermeta");
         othermeta.setAttribute("content", family);
         othermeta.setAttribute("name", "pageid");
         metadata.appendChild(othermeta);
         prolog.appendChild(metadata);
-        topic.appendChild(prolog);
+        reference.appendChild(prolog);
 
         final Element body = xml.createElement("refbody");
         body.setAttribute("outputclass", "subscription");
 
+        Map<String, Map<String, List<Param>>> parametersWithUInfo2 = componentDescription.getParametersWithUInfo();
+
+        // Map<String, Map<String, SortedSet<UIParam>>> parametersWithUInfo =
+        // componentDescription.getParametersWithUInfo();
+
+        generateConfigurationSection(xml, body, family, name, reference.getAttribute("id"),
+                parametersWithUInfo2.get("datastore"), "Datastore");
```

Comment:
Officially it is seen as "Configuration" in the ui.
Just ensuring it is intended to be datastore there.

Reply:
Replaced by Connection (also in IDs)